Java/PlatinumGenomes-variant-transformation/pom.xml

```diff
@@ -11,7 +11,7 @@
     <dependency>
       <groupId>com.google.cloud.genomics</groupId>
       <artifactId>google-genomics-dataflow</artifactId>
-      <version>v1beta2-0.15</version>
+      <version>v1beta2-0.16-SNAPSHOT</version>
```

Comment:
Yes, for now.  Since its in a branch, I think its okay to leave it pointing to unreleased code.  Let me know if you disagree.